src/file.js

```diff
@@ -21,3 +21,11 @@ module.exports.getFragmentsContent = function(fragmentsFiles) {
     return fs.readFileSync(file, "utf8");
   });
 };
+
+module.exports.delFragmentsFiles = function(fragmentsFolder) {
+  let dir = fs.readdirSync(fragmentsFolder);
+
+  dir.forEach(file => {
```

Comment:
We can't delete all files from fragment folder. To git track a folder, we need to keep an file inside there, like `.gitkeep`.

This function may receive a list of files result from `getFragmentsFilesByFragmentType` for example.

Reply:
I think that is a better way to do it just ignoring the hidden files with a `.filter()` on the `.readdirSync()` in terms of using that function. 

If we receive a list of files from the `getFragmentsFilesByFragmentType` we'll need to make a loop between every registered type in order to delete the entire `fragments` folder. (lookup `be92746`)

What do you think?

Reply:
I totally disagree this approach. 

The propose of this function is `deleteFragmentsFiles`, not `clearFragmentsFolder`. 

We can't assume who use this plugin only put fragments files in this folder. I can put there the template file for `news-fragments`, an text or md file explaining the usage and propose of folder, etc.

So, it's not a problem iterate and delete only a list of files received in this function.

Reply:
It really makes more sense when we think that way. Nice thought, I totally agree with that. 
I've did some changes to work properly on this way, check `6507a53`.